Automattic/wp-calypso
# Jetpack connect: New example images

fix https://github.com/Automattic/wp-calypso/issues/5176#issuecomment-220587710

This PR updates the images on the examples provided in jetpack connect to be more accurate

![image](https://cloud.githubusercontent.com/assets/1554855/15432073/efa6d8ce-1eac-11e6-8218-9ee65179b308.png)

How to test:
=========
1. Go to http://calypso.localhost:3000/jetpack/connect
2. Enter a site that doesn't have jetpack installed
3. You should see the new images in the examples \o/

@rickybanister @folletto
